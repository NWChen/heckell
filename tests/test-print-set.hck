let S1 : char set;
S1 = { 'c', 'h', 'a', 'n' };
print_set(S1);

let S2: real set;
S2 = {1.3, 4.096, 0.989, 1.3};
print_set(S2);

print_set({1,3,3,89});
print_set({true, false});