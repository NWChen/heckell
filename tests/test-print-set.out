{ c h a n }
{ 1.300000 4.096000 0.989000 }
{ 1 3 89 }
{ true }
